README.md

```diff
@@ -94,6 +94,43 @@ First, install it: `npm install --save-dev aliasify`
 }
 ```
 
+## Usage with Rollup
+
+Using `preact-compat` with rollup requires [rollup-plugin-alias](http://npm.im/rollup-plugin-alias), along with a few other plugins
+
+First, install it: `npm install --save-dev rollup-plugin-alias rollup-plugin-buble rollup-plugin-commonjs rollup-plugin-node-resolve rollup-plugin-replace`
+
+... then in your `rollup.config.js`, configure the alias plugin to point to `react` and `react-dom`, along with the rest of the plugins:
+
+```js
+import alias from "rollup-plugin-alias";
+import buble from "rollup-plugin-buble";
+import resolve from "rollup-plugin-node-resolve";
+import replace from "rollup-plugin-replace";
+import commonjs from "rollup-plugin-commonjs";
+// ...
+
+export default {
+    // ...
+    plugins: [
+        // ...
+        resolve({
+            jsnext: true // preact
+        }),
+        alias({
```

Comment:
Unrelated to this docs PR, but do you think it might be useful for `preact-compat` to include an export that auto-generates this?

```js
import alias from 'rollup-plugin-alias';
import aliases from 'preact-compat/aliases';

export default {
  // ...
  plugins: [
    alias(aliases)
  ]
};
```